lib/api/controllers/routerController.js

```diff
@@ -54,7 +54,7 @@ module.exports = function RouterController (kuzzle) {
       executeFromRest.call(kuzzle, params, request, response);
     });
 
-    api.get('/_getStats', function (request, response) {
+    api.post('/_getStats', function (request, response) {
```

Comment:
I'd better keep a GET method and add the timestamp as a URI parameters :
api.get('/_getStats/:timestamp', ....

So we can get the stats from a timestamp with the request :
GET http://kuzzle:xxxx/api/_getStats/12341564897